app/services/fastly_vcl/whitelisted_params.rb

```diff
@@ -0,0 +1,65 @@
+module FastlyVCL
+  # Handles updates to our VCL snippet on Fastly that whitelists params
+  class WhitelistedParams
+    VCL_DELIMITER_START = "^(".freeze
+    VCL_DELIMITER_END = ")$".freeze
+    SNIPPET_NAME = "Whitelist certain querystring parameters".freeze
```

Comment:
`query string`

Reply:
This is how it's named in Fastly already, I didn't want to change it.